src/fusion-rates-gamow.md

```diff
@@ -0,0 +1,861 @@
+
+For the $D_2$ molecule:
+- The reacting volume is just the volume of the nucleus $v_{nuc}$
+- The rate at which particles in this volume react is given by $\gamma$
+- The density of nuclei is calculated by the inverse the volume occupied by a single molecule $1/v_{mol}$.
+- The effective density of nuclei in the reacting volume is $e^{-2G}/v_{mol}$ due to the repulsion of the nuclei.
+
+This results in a fusion rate of:
+
+$$\Gamma = \frac{v_{nuc}}{v_{mol}}e^{-2G} \gamma$$
+
+where one can interpret $\frac{v_{nuc}}{v_{mol}}e^{-2G}$ as the probability for the particles to have traversed the Coulomb barrier. One might also have arrived at this probability from an intuitive consideration of a particle trapped in a box of volume $v_{mol}$ moving around randomly and seeking to land in a target volume $v_{mol}$ but hindered by a factor $e^{-2G}$.
+
+Now we've got an expression for the fusion rate, we just have to calculate all the relevant pieces. Let's start with constructing the interatomic potential that's needed for the Gamow factor.
+
+
+## Potentials
+
+<!-- #region -->
+In the calculation of the Gamow factor, the interatomic potential needs to be augmented with the centripetal potential to create an effective potential:
+
+$$V_{\rm eff}(r) = V(r) + \frac{L(L+1)\hbar^2}{2\mu r^2}$$
+
+
+For a $\rm D_2$ molecule, the interatomic potential $V(r)$ consists of 2 parts - the nuclear and molecular potentials:
+
+$$V(r) = V_{\rm nuc}^{S,L}(r) + V_{\rm mol}(r)$$
+
+
+<!-- #endregion -->
+
+### Nuclear potential
+
+We use the Woods-Saxon nuclear potential (in MeV):
+
+$$V_{\rm nuc}^{S,L}(r) ~=~ {V_0 \over 1 + e^{(r - r_S) / a_S}}$$
+
+
+```python
+# The nuclear Woods-Saxon nuclear potential
+def V_nuc(r, V0, r_S, a_S):
+    r_safe = np.minimum(r, 100) # Prevents overflow errors in the exponential when r is in the pm range
+    return V0 / (1 + np.exp((r_safe - r_S) / a_S))
+```
+
+The parameters depend on the total spin ($S$) and orbital ($L$) angular momenta of the nuclei as seen below and can be found in [Tomusiak et.al](http://dx.doi.org/10.1103/physrevc.52.1963).
+
+| State | $V_0$ (MeV) | $r_s$ (fm) | $a_s$ (fm) |
+|-------|-------------|------------|------------|
+| $^1S$ | -74.0       | 1.70       | 0.90       |
+| $^5S$ | -15.5       | 3.59       | 0.81       |
+| $^3P$ | -13.5       | 5.04       | 0.79       |
+| $^5D$ | -15.5       | 3.59       | 0.81       |
+
+```python
+# Parameters for nuclear potential depend on the state
+states = [
+    {"state": r"$^1S$: L=0, S=0", "L":0, "S":0, "V0": -74.0, "r_S": 1.70, "a_S": 0.90},
+    {"state": r"$^5S$: L=0, S=2", "L":0, "S":2, "V0": -15.5, "r_S": 3.59, "a_S": 0.81},
+    {"state": r"$^3P$: L=1, S=1", "L":1, "S":1, "V0": -13.5, "r_S": 5.04, "a_S": 0.79},
+    {"state": r"$^5D$: L=2, S=2", "L":2, "S":2, "V0": -15.5, "r_S": 3.59, "a_S": 0.81},
+]
+```
+
+We can then see that the attractive nuclear potentials become negligible within about 10 fm.
```

Comment:
Is this correct? The wording here is confusing me a bit. Is meant "negligible beyond about 10 fm"?